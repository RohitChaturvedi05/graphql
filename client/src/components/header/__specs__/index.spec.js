import React from 'react'
import TablePagination from '@mui/material/TablePagination'
import { mount, shallow } from 'enzyme'
import getAllUsers from '../../../services/get-all-users'
import Headers from '../index'


describe('Header', () => {
    it('should render Headers with correct properties', () => {
        const wrapper = shallow(<Headers />)
        expect(wrapper).toMatchSnapshot()
    })
})