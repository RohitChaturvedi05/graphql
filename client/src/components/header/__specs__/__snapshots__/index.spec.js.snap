// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Headers with correct properties 1`] = `ShallowWrapper {}`;
